examples/DotNetCore/EventHub/Controllers/HomeController.cs

```diff
@@ -0,0 +1,169 @@
+﻿using System.Diagnostics;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+using SimpleAppConfigEventHub.Models;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Storage.Blobs;
+using Azure.Messaging.EventHubs.Consumer;
+using Azure.Messaging.EventHubs.Processor;
+using Newtonsoft.Json;
+using Azure.Messaging.EventHubs;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration;
+
+namespace SimpleAppConfigEventHub.Controllers
+{
+    public class HomeController : Controller
```

Comment:
>HomeController [](start = 17, length = 14)

There's a lot going on in this controller. How can we make it so that it knows nothing about event hub?

Can we come up with a service that exposes events that the home controller would care about. That service can abstract event hub

Reply:
Moved it to a different service.